update.py

```diff
@@ -160,29 +160,54 @@ def commit_changes(githubtoken):
 help = Show this help
 """
 
-def update_github_releases_for_assets(githubtoken, include_prerelease=False, per_page=100, release_limit=50):
+def update_github_releases(githubtoken, asset_id=None, include_prerelease=False, per_page=100, release_limit=50):
+    """Update GitHub releases for all assets or a single asset.
+
+    When asset_id is provided, only that asset JSON is processed.
+    Otherwise, all JSON files under assets/ are updated.
+    """
     if githubtoken is None:
         print("No GitHub token specified")
         sys.exit(1)
 
-    print("Update releases for assets")
-    for filename in find_files("assets", "*.json"):
-        print("Getting latest release for %s" % filename)
+    # Build file list
+    if asset_id:
+        filename = os.path.join("assets", asset_id + ".json")
+        if not os.path.exists(filename):
+            print("Asset JSON not found: %s" % filename)
+            sys.exit(1)
+        files = [filename]
+        print("Update releases for asset %s" % asset_id)
+    else:
+        files = find_files("assets", "*.json")
+        print("Update releases for assets")
+
+    for filename in files:
+        if not asset_id:
+            print("Getting latest release for %s" % filename)
+
         asset = read_as_json(filename)
         if not asset:
             print("...error!")
+            if asset_id:
```

Comment:
Could you please remind me why a singel asset vs all assets are treated differently?